google-ads-examples/src/main/java/com/google/ads/googleads/examples/extensions/AddPrices.java

```diff
@@ -0,0 +1,237 @@
+    } catch (FileNotFoundException fnfe) {
+      System.err.printf(
+          "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", fnfe);
+      return;
+    } catch (IOException ioe) {
+      System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+      return;
+    }
+
+    try {
+      new AddPrices().runExample(googleAdsClient, params.customerId, params.campaignId);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+    }
+  }
+
+  /** Runs the example. */
+  private void runExample(GoogleAdsClient googleAdsClient, long customerId, long campaignId) {
+    // Creates an extension feed item as price.
+    String extensionFeedItemResource =
+        createExtensionFeedItem(googleAdsClient, customerId, campaignId);
+
+    // Creates a customer extension setting using the previously created extension
+    // feed item. This associates the price extension to your account.
+    CustomerExtensionSetting setting =
+        CustomerExtensionSetting.newBuilder()
+            .setExtensionType(ExtensionType.PRICE)
+            .addExtensionFeedItems(StringValue.of(extensionFeedItemResource))
+            .build();
+
+    // Issues a mutate request to add the customer extension setting and print its information.
+    CustomerExtensionSettingOperation operation =
+        CustomerExtensionSettingOperation.newBuilder().setCreate(setting).build();
+    try (CustomerExtensionSettingServiceClient client =
+        googleAdsClient.getLatestVersion().createCustomerExtensionSettingServiceClient()) {
+      MutateCustomerExtensionSettingsResponse response =
+          client.mutateCustomerExtensionSettings(
+              String.valueOf(customerId), Arrays.asList(operation));
+      System.out.println(
+          "Created customer extension setting with resource name: "
+              + response.getResults(0).getResourceName());
+    }
+  }
+
+  /** Creates an extension feed item for price extension. */
+  private String createExtensionFeedItem(
+      GoogleAdsClient googleAdsClient, long customerId, long campaignId) {
+    // Creates the price extension feed item.
+    PriceFeedItem priceFeedItem =
+        PriceFeedItem.newBuilder()
+            .setType(PriceExtensionType.SERVICES)
+            .setPriceQualifier(PriceExtensionPriceQualifier.FROM)
```

Comment:
There is a comment in the PHP example that this is optional. Was that intentional to omit it here?

Reply:
Nope, added.